Fix CI

HuKeping opened the pull request:
The help message of cpu quota is over 80 chars, so it failed the
TestHelpTextVerify test.

Error as below:

```
FAIL: docker_cli_help_test.go:14: DockerSuite.TestHelpTextVerify

docker_cli_help_test.go:115:
    c.Fatalf("Help for %q is too long(%d chars):\n%s", cmd,
        len(line), line)
... Error: Help for "create" is too long(81 chars):
  --cpu-quota=0               Limit the CPU CFS (Completely Fair Scheduler) quota
```

Signed-off-by: Hu Keping hukeping@huawei.com

duglin commented:
How did that line ever make it into master in the first place and why isn't janky failing on it?  very odd...

duglin commented:
@HuKeping you have one more space in that help text than I do, which I guess is why its failing for you. 
In looking at the `docker create --help` output I don't see anything "variable" (like a long $HOME value) that would cause this. I think we need to figure out why this is only failing on your machine first.

RickWieman commented:
If it *does* need a change of only one or two characters, why not remove the acronym and keep the full part? I don't think many people know the acronym CFS (at least I didn't).

cpuguy83 commented:
I would just make it `Set CPU CFS quota`

duglin commented:
I'm not as interested in the specific text right now - we can easily shrink it. I'm more curious to know why this fails on his machine but not mine or janky.

thaJeztah commented:
@duglin diffent code page?

duglin commented:
Perhaps but it seems like if it were the code page then other lines would be too long since others are close as well.  Plus, just one extra space, nothing else, very odd.

HuKeping committed:
Fix CI

The help message of cpu quota is over 80 chars, so it failed the
TestHelpTextVerify test.

Error as below:

```
FAIL: docker_cli_help_test.go:14: DockerSuite.TestHelpTextVerify

docker_cli_help_test.go:115:
    c.Fatalf("Help for %q is too long(%d chars):\n%s", cmd,
        len(line), line)
... Error: Help for "create" is too long(81 chars):
  --cpu-quota=0  Limit the CPU CFS (Completely Fair Scheduler) quota
```

Signed-off-by: Hu Keping <hukeping@huawei.com>

jfrazelle commented:
LGTM

calavera commented:
LGTM

duglin commented:
Maintainers: please don't merge this until we understand WHY its failing.

HuKeping commented:
![81](https://cloud.githubusercontent.com/assets/9958621/7289079/79b7d43c-e99d-11e4-802f-7e31b272562a.PNG)

HuKeping commented:
hi @duglin  This is from `docker help run`, it is odd indeed, see the 13 line

duglin commented:
where does the `--oom-kill-disable` come from?
I don't see that and I'm on master.

duglin commented:
I think that's why your first column is wider than everyone else's.

HuKeping commented:
oh I see!! so sorry for the mistake, I'll close this.
